### ISSUE TITLE
CodePath Issue

### ISSUE THREAD

**Issue by nisargap** [LectureGenius/codepath-project#1]
/cc @codepathreview

**Comment by minhkhang1795** [comment]
:+1: Great to see your GitHub repository setup and user stories defined! The user stories are a key piece of the group project. They help you prioritize the most important parts of the app to build. It's particularly important that you've split the user stories into "Required" and "Optional" stories so your team can focus on the most important aspects of the app first and then add in the optionals as time permits toward the end of the project.

Next week we'll move on to the next step of the group project by doing something called wireframing, which is creating a blueprint for the app.

**Comment by minhkhang1795** [comment]
Wait a minute! @nisargap you should be in a different group, shouldn't you?!

**Comment by nisargap** [comment]
Yeah, I'm in Routru but I sent in the issue that week for all the groups (I'm a student leader) because the groups were unaware that they had to submit an issue that week. It shouldn't be problem anymore (I won't be submitting anymore issues on their behalf). Thanks! :+1:
